torch/standalone/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.27 FATAL_ERROR)
```

Comment:
I would not cmake it a `CMakeLists.txt`, but rather `configure_macros.cmake` or something

`CMakeLists.txt` implies that this is a top-level project file which could be used on its own, which is clearly not the case here

Reply:
> CMakeLists.txt implies that this is a top-level project file which could be used on its own, which is clearly not the case here

I don't think that's true. Not every subdirectory has to be its own project. https://cmake.org/cmake/help/latest/command/add_subdirectory.html